.github/workflows/integrations_tests.yml

```diff
@@ -0,0 +1,94 @@
+name: integration tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to test on'
+        required: true
+      pytorch_nightly:
+        description: 'Whether to test integration tests (diffusers + transformers)'
+        required: false
+        default: false
```

Comment:
This parameter is not being used, is it? Also, description and parameter name don't fit each other.

Reply:
Correct, I have simply removed it here